Versions/Common/BeStride_Mount.lua

```diff
@@ -219,28 +219,35 @@ function BeStride_Mount:DemonHunterGlide()
 end
 
 function BeStride_Mount:Druid()
-	local travelForm, flightForm = 783, 783 -- 3 in 1 travel form
+	local travelForm, flightForm = BeStride_Constants.spells.druid.travelform, BeStride_Constants.spells.druid.flightform
 
 	if GetUnitSpeed("player") ~= 0 then
-		return self:MountSpell(SpellToName(783))
+		return self:MountSpell(SpellToName(travelForm))
 	elseif BeStride:DruidFlyingMTFF() or IsFalling() or IsFlying() or GetShapeshiftForm() == 3 then
```

Comment:
Just a little suggestion, move this condition before the speed check. 
Otherwise, if you decide to buff while in flight, you better pull your spell book fast if you don't want to end up flat on the ground. :)

Reply:
On a more global note, I'm not convinced on this speed condition at all. I can't imagine a situation where I would prefer the travel form when flying is possible. 
It should default to travel form while moving only if flying is not possible (not learned, combat, non-flyable zone...)

Reply:
Maybe GetUnitSpeed() and not IsFlying() and not IsFalling()

Might also need to be a new conditional along the lines of:

IsFalling() and GetUnitSpeed() and DruidCanFlightForm() or something

Reply:
As suggested by @Sharptsa I removed this function as it's not used anywhere.